src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1950,5 +1950,7 @@ public void GetPartsOfCastExpression(SyntaxNode node, out SyntaxNode type, out S
 
             return null;
         }
+
+        public bool IsLambdaBody(SyntaxNode node) => node.Parent != null && node.Parent is LambdaExpressionSyntax lambdaNode && lambdaNode.Body == node;
```

Comment:
`node.Parent != null` check is redundant.  it's implied by `node.Parent is LambdaExpression`.